### ISSUE TITLE
figma: search experience wireframes

### ISSUE THREAD

**Issue by arugyani** [arugyani/shopIQ#2]
# [UI Wireframes](https://www.figma.com/file/2yDlIN9SPqYM03XHI62soi/Senior-Project---hcrUI?type=design&node-id=0%3A1&mode=design&t=740iA2Ye73Zt7HzZ-1)
The goal of this issue is to create wireframe layouts for the frontend UI of the project.
Considerations for user experience, accessibility, and aesthetic visual design should be made.

TODO:
- [x]  Determine required pages
- [x] Create box-layout for each page.
